docs/articles/expensify-classic/insights-and-custom-reporting/Fringe-Benefits.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Fringe Benefits
+description: How to track your Fringe Benefits
+---
+# Overview 
+If you’re looking to track and report expense data to calculate Fringe Benefits Tax (FBT), you can use Expensify’s special workflow that allows you to capture extra information and use a template to export to a spreadsheet.
+
+# How to set up Fringe Benefit Tax
+
+## Add Attendee Count Tags 
+First, you’ll need to add these exact two tags to your Workspace: 
```

Comment:
```suggestion
First, you’ll need to add these two tags to your Workspace: 
```
Adding exact here can be confusing because it could mean that you should only have 2 tags in the Workspace. The following paragraph explains that they need to be added exactly as written so I think that's adequate.